File: apps/motoboy-pwa/src/locationActionLabels.ts
export function availabilityActionLabel(available: boolean): string {
  return available ? "Parar corridas" : "Comecar corridas";
}

export function availabilityShortActionLabel(): string {
  return "Corridas";
}

export function automaticLocationActionLabel(trackingEnabled: boolean): string {
  return trackingEnabled ? "Parar GPS ao vivo" : "Ligar GPS ao vivo";
}

export function sendLocationActionLabel(loading: boolean): string {
  return loading ? "Enviando..." : "Enviar minha posicao";
}

File: apps/motoboy/app/src/main/kotlin/com/drogsantoantonio/farmadelivery/presentation/ui/screens/LocationActionLabels.kt
package com.drogsantoantonio.farmadelivery.presentation.ui.screens

fun availabilityActionLabel(available: Boolean, loading: Boolean): String {
  if (loading) return "Salvando..."
  return if (available) "Parar corridas" else "Comecar corridas"
}

fun automaticLocationActionLabel(trackingEnabled: Boolean): String {
  return if (trackingEnabled) "Parar GPS ao vivo" else "Ligar GPS ao vivo"
}

fun sendLocationActionLabel(loading: Boolean): String {
  return if (loading) "Enviando..." else "Enviar minha posicao"
}
